server/gorm/client.go

```diff
@@ -213,7 +211,7 @@ func (c *Client) Put(ctx context.Context, k storage.Key, v interface{}) (storage
 	}
 	c.db.Transaction(
 		func(tx *gorm.DB) error {
-			rowsAffected := tx.Model(v).Where("key = ?", k.(*Key).Name).Updates(v).RowsAffected
+			rowsAffected := tx.Model(v).Select("*").Where("key = ?", k.(*Key).Name).Updates(v).RowsAffected
```

Comment:
Looks good. It might help to link to https://gorm.io/docs/update.html#Update-Selected-Fields for background on this.